docs/configure

```diff
@@ -77,8 +77,9 @@ ALIAS_DOMAINS = SELECT DISTINCT alias FROM domainalias WHERE alias = '${quote_my
 domainlist local_domains = @ : example.org : ${lookup mysql{VIRTUAL_DOMAINS}} : ${lookup mysql{ALIAS_DOMAINS}}
 domainlist relay_to_domains = ${lookup mysql{RELAY_DOMAINS}}
 hostlist   relay_from_hosts = localhost : @ : 192.168.0.0/24
-trusted_users = avleen:www
+trusted_users = vexim:www-data
```

Comment:
That there might be problems with cronjobs, quota-messages, ... if the mail addresses have not been configured properly. If we change this, we should keep a document with these changes that allows you to check possible problems.